### ISSUE TITLE
Yolov5 support

### ISSUE THREAD

**Issue by iamsb1896** [shaqian/flutter_realtime_detection#35]
Does it support other tflite models such as Yolov5 ?

**Comment by HunarAA** [comment]
Did you find the answer for your question? I have trained yolov5 model and converted to tflite then embedded in this code but the app will automatically closes when I choose my model.

**Comment by iamsb1896** [comment]
Not really. I just build both iOS and Android using Native Language. For Android I converted model to TFLite and for iOS Pytorch Lite

**Comment by HunarAA** [comment]
What tools did you use? How do you convert your yolov5 model into tflite? because I think the problem is with my model conversion?

**Comment by dnth** [comment]
@HunarAA there is an export script from the YOLOv5 repo that converts the model into tflite. 

Btw, did anyone get YOLOv5 working with this repo?

**Comment by HunarAA** [comment]
till now I am not able to get it work. The app closes directly when I use yolov5 tflite model

**Comment by fransay** [comment]
@HunarAA consider rebuilding your model with ssd mobileNET, it seems there is no export script for yolov5 yet
